libs/MeshKernelApi/src/MeshKernel.cpp

```diff
@@ -2290,7 +2290,7 @@ namespace meshkernelapi
         return lastExitCode;
     }
 
-    MKERNEL_API int mkernel_mesh2d_connect_meshes(int meshKernelId, const Mesh2D& mesh2d)
+    MKERNEL_API int mkernel_mesh2d_connect_meshes(int meshKernelId, const Mesh2D& mesh2d, double searchFraction)
```

Comment:
Change in signature of API function, must adapt MKPy, MKNet and GEP.

Reply:
MKNet and GEP do not support this functionality yet. Adapting MKPy will be done in this [PR](https://github.com/Deltares/MeshKernelPy/pull/93).